src/SFA.DAS.EmployerRequestApprenticeTraining.Web/Validators/EmployerRequest/EnterApprenticesEmployerRequestViewModelValidator.cs

```diff
@@ -30,6 +32,11 @@ private static bool BeValidNumber(string numberOfApprentices)
             return int.TryParse(numberOfApprentices, out _);
         }
 
+        private static bool BeWholeNumber(string numberOfApprentices)
+        {
+            return string.IsNullOrEmpty(numberOfApprentices) || !numberOfApprentices.Contains('.');
+        }
+
         private static bool BeWithinValidRange(string numberOfApprentices)
```

Comment:
A bit of a tangent here - I was thinking if there any other characters we need to filter out, so I looked at the GDS guidelines for numeric fields and it recommended adding inputmode="numeric" to the input field if you're asking for whole numbers

Reply:
I will do that